tests/attestation_plugin/plugin/mock_attester.h

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Maybe provide a description of what this test is doing?

Reply:
I added a description.